scripts/run-test-suite

```diff
@@ -5,6 +5,8 @@ RIOT_PATTERN=${1}
 DDTRACE_FLAG=$([ -v _CI_DD_API_KEY ] && echo '--ddtrace')
 COVERAGE_FLAG=$([[ "${2:-false}" == false ]] && echo '--no-cov')
 DDTEST_CMD=$([[ ${3} == "1" ]] && echo "./scripts/ddtest")
+# temporary workaround for pytest issues in CI jobs
+DDTRACE_FLAG=$([[ "${4:-false}" == true ]] && echo '--no-ddtrace')
```

Comment:
Could you explain why we need this in the description?

Reply:
This was due to the pytest incompatibility that's getting worked on in this PR: https://github.com/DataDog/dd-trace-py/pull/8357

If it gets merged tomorrow, I'll remove this workaround and comment. If it looks like it's gonna take longer, I'll just merge this with this workaround and once the other PR is merged, I'll remove this workaround.

Reply:
@erikayasuda , I would pin `pytest` to `~=7.0` in the relevant environments rather than disable the `ddtrace` plugin.

Reply:
Removed the workaround instead since it looks like the other PR merged. If there are still issues I'll pin `pytest` to `~=7.0` for the `django` test suites